action/protocol/poll/consortium.go

```diff
@@ -25,6 +25,7 @@ import (
 var (
 	consortiumCommitteeContractCreator = address.ZeroAddress
 	consortiumCommitteeContractNonce   = uint64(0)
+	consortiumCommitteeContractHash, _ = hash.HexStringToHash256("00636f6e736f727469756d436f6d6d6974746565436f6e747261637448617368")
```

Comment:
`00636f6e736f727469756d436f6d6d6974746565436f6e747261637448617368` - what is this magic string?

Reply:
this is the hex of "consortiumCommitteeContractHash"